src/components/Login.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import LoginLocal from './LoginLocal'
+import RegisterNewUser from './RegisterNewUser'
+export default class Login extends React.Component{
+	render() {
+		return(
+			<div className="wrapper">
+				<LoginLocal getuser={this.props.getuser}/>
```

Comment:
You don't seem to be passing a getUser function to the Login component so there wouldn't be anything to pass down to LoginLocal. Is this a holdover from PeTinder? I'm not sure all of that code still applies now that we are using Redux.